### ISSUE TITLE
Pods snippet

### ISSUE THREAD

**Issue by doggy8088** [ipedrazas/kubernetes-snippets#4]
I can't found any `pods` snippet in this extension.  Are you considering to put this onto your extension?

**Comment by ipedrazas** [comment]
yes, I'm going to review completely the extension in the next weeks.

**Comment by mkamrani** [comment]
This is a good extension, just the pods is a major part missing.
